examples/io/viewers/viewer3D-10-interaction.cpp

```diff
@@ -50,6 +53,11 @@ struct BigData
   std::map< DGtal::int32_t, Z3i::SCell > cells;
 };
 
+struct BigDataV
```

Comment:
Suggesting BigDataVoxels instead of BigDataV (and BigDataCells instead of BigData).